yb-voyager/src/tgtdb/oracle.go

```diff
@@ -616,6 +685,17 @@ func (tdb *TargetOracleDB) getConnectionString(tconf *TargetConf) string {
 	return connectString
 }
 
+func (tdb *TargetOracleDB) GetImportStatsMetaInfo(migrationUUID uuid.UUID) (int64, int64, int64, error) {
```

Comment:
Same comments that were provided for the YB implementation apply here as well.

We're having significant code duplication. I will have to refactor the `TargetDB` design to avoid it.

Reply:
Yes Agreed!, a few functions are completely duplicated it's just we have to keep them in both places.